doc/en/components/navigation-drawer.md

```diff
@@ -0,0 +1,237 @@
+# $Platform$ Navigation Drawer Overview
+
+The Infragistics $Platform$ [IgcNavDrawerComponent]({environment:wcApiUrl}/classes/igcnavdrawercomponent.html) is a WebComponent that provides side navigation. It can be expanded or collapsed within the content. A mini version provides quick access to navigation even when closed. Its content is completely customizable while also providing default menu item styling.
```

Comment:
I would suggest to use **Navigation Drawer** here instead the component class name since the first paragraph is very important for the SEO. You can use the component class name in the **Usage** section below.